inspire/base/static/js/search/form.js

```diff
@@ -32,6 +32,10 @@ define([
     return true; // ensure form still submits
   })
 
+  $("button[name=action_search]").on("click", function(){
```

Comment:
yes it counts as click.

2015-11-24 13:32 GMT+01:00 Javier Martin Montull <notifications@github.com>:

> In inspire/base/static/js/search/form.js
> <https://github.com/inspirehep/inspire-next/pull/516#discussion_r45728768>
> :
>
> > @@ -32,6 +32,10 @@ define([
> >      return true; // ensure form still submits
> >    })
> >
> > +  $("button[name=action_search]").on("click", function(){
>
> When you add a search query to the search bar and press Enter, does it
> count as click ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/516/files#r45728768>.
>